.github/workflows/_build_artifacts.yml

```diff
@@ -0,0 +1,326 @@
+name: Build Artifacts
```

Comment:
Does the leading underscore in the file name mean that this is only called by other workflows, and isn't top-level?

Reply:
It's a convention I think @AndrewDryga started, so I kept it for this refactor too.

Reply:
@ReactorScram I just tried to separate the workflow that we do run from helper files like that we just include because we have a lot of them and GitHub actions can't use subfolders.

Reply:
Makes sense, I just wanted to be sure my guess was right